lib/kino/table.ex

```diff
@@ -104,6 +114,19 @@ defmodule Kino.Table do
     {:noreply, broadcast_update(ctx)}
   end
 
+  def handle_event("download", %{"format" => format}, ctx) do
+    download_data = ctx.assigns.module.export_data(ctx.assigns.state, String.to_atom(format))
+
+    send_event(ctx, ctx.origin, "download_content", %{
+      "data" => download_data.data,
```

Comment:
Since data can be arbitrary binary, we should use [binary payload](https://hexdocs.pm/kino/Kino.JS.Live.html#module-binary-payloads) :)